content/english/security-fundamentals/security-vs-privacy.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Security vs Privacy"
+description: ""
+draft: false
+weight: 3
+---
+
+Security & privacy are often used interchangeably by many people, but it is important to understand how these two ideas are different. Security refers to protection against the unauthorized access of data regardless of the content. Privacy refers to the right for consumers to be free from unwanted attention or surveillance. Privacy is generally concerned with Personally Identifiable Information (PII). Some examples of PII are email addresses, phone numbers, addresses, birthdays, credit card details, and even your fingerprint.  We put security controls in place to limit who can access information, and we put privacy controls in place to limit the type of information that is accessible in certain places 
+
+
```

Comment:
Not needed for first draft, but a nice picture would make this section more presentable.

Reply:
Does this look fine @ozhang22:
![Security vs  Privacy](https://user-images.githubusercontent.com/96280466/182464751-a81d60ea-c62a-4ab9-a543-bb4cee7341da.png)
Please let me know if you have any suggestions/feedback on how to improve it.

Reply:
If you want to edit it yourself, please ask me for the link (I created this on Canva).

Reply:
I like the picture! I would still like the link to change the wording in some spots and to maybe adjust the colors to adhere to the Nuevo colors.

Reply:
Okay I went ahead and made the changes to the file, it would just need to be exported again. @ozhang22 we should probably loop in Izzy before adding it in.

Reply:
Sounds good, I've attached the exported image. Please let me know if you need me to export it again after making further changes.
![Protection of personal information](https://user-images.githubusercontent.com/96280466/183222817-72a3f354-dfdd-4839-8ddf-36a155540cf7.png)

Reply:
I made one last modification so that the font matches the other Nuevo graphics. Here is the final one I added in.

![Protection of personal information](https://user-images.githubusercontent.com/33306040/183225454-d1724549-3eed-41b8-8808-ba9558d9fbba.png)

Reply:
Looks good!